pkg/domain/entities/manifest.go

```diff
@@ -36,7 +36,7 @@ type ManifestAddOptions struct {
 // ManifestAnnotateOptions provides model for annotating manifest list
 type ManifestAnnotateOptions struct {
 	// Annotation to add to manifest list
-	Annotation []string `json:"annotation" schema:"annotation"`
```

Comment:
@Luap99 This'd be a breaking change, right?

Reply:
This should be OK. `pkg/domain/entities` is an internal package.

Reply:
No this is breaking the rest API, the structs in `pkg/domain/entities` are sometimes used in the API to unmarshal the query.
see https://github.com/containers/podman/blob/226977fafbad97c1db4618de8517151d3242a1c1/pkg/domain/entities/manifest.go#L18-L19 and https://github.com/containers/podman/blob/226977fafbad97c1db4618de8517151d3242a1c1/pkg/domain/entities/manifest.go#L63-L65
which is then parsed in https://github.com/containers/podman/blob/226977fafbad97c1db4618de8517151d3242a1c1/pkg/api/handlers/libpod/manifests.go#L435-L436

Reply:
Thank you for your review.
I'll think about other methods.

Reply:
Add an `Annotations` field while keeping an `Annotate` Field.